src/validators/ClassValidator.ts

```diff
@@ -223,6 +224,24 @@ export class BsClassValidator {
                                 range: member.range
                             });
                         }
+
+                        //child member has different visiblity
+                        if (
+                            //is a method
+                            isClassMethodStatement(member) &&
```

Comment:
Why only class methods? Fields should also be included in this. TypeScript applies this restriction to both methods and fields.

Reply:
I did this as we don't support overriding field yet, do we?